### PR TITLE
Add Haqq networks

### PR DESCRIPTION
**Pull request by Hirama**
Please add Haqq Mainnet and HaqqTestnet:

https://github.com/safe-global/safe-deployments/pull/335

Thank you!

### CODE REVIEW

**Comment by Hirama** [comment]
I have read the CLA Document and I hereby sign the CLA

File: gnosis/safe/addresses.py [hunk 1]
```diff
@@ -655,6 +655,18 @@
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 338471, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 338472, "1.3.0"),
     ],
+    EthereumNetwork.HAQQ_NETWORK: [
+        ("0x29fcB43b46531BcA003ddC8FCB67FFE91900C762", 9054796, "1.4.1+L2"),
+        ("0x41675C099F32341bf84BFc5382aF534df5C7461a", 9054798, "1.4.1"),
+        ("0xfb1bffC9d739B8D520DaF37dF666da4C687191EA", 422246, "1.3.0+L2"),
```
Comment (moisses89) [inline gnosis/safe/addresses.py:661]: Address for L2 is different on Safe Deployments. https://github.com/safe-global/safe-deployments/blob/main/src/assets/v1.3.0/gnosis_safe_l2.json#L124

**Review by moisses89 — CHANGES_REQUESTED** [review]
Thanks for your contribution, could you check the addresses?

**Comment by Hirama** [comment]
@moisses89 thanks for comments! I have applied fixes.

**Review by moisses89 — APPROVED** [review]

**Comment by moisses89** [comment]
@Hirama could you fix the conflicts, ping me when ready.

**Comment by Hirama** [comment]
@moisses89 resolved, please check

**Comment by Hirama** [comment]
@moisses89 resolved, please check

**Pull request merged**